### PR TITLE
Style fixes in cosmos json files

### PR DESCRIPTION
**Pull request by mbdcos**

### CODE REVIEW

**Comment by mbdcos** [comment]
Please point others if there is any, or you can push a commit this PR

**Comment by elezar** [comment]
Thanks @mbdcos. I applied automatic json formatting to the files edited here (using the VSCode JSON Tools plugin) and pushed a commit.

**Comment by elezar** [comment]
(for the record, we could consider using: 
```
cat config.json | jq
``` 
or
```
cat config.json | python -m json.tool
```
to do this automatically.

File: frameworks/helloworld/universe/config.json [hunk 1]
```diff
@@ -1,158 +1,158 @@
 {
-  "type":"object",
-    "properties":{
```
Comment (elezar) [inline frameworks/helloworld/universe/config.json]: For the record, the `"properties"` element was incorrectly indented here. Hence the large looking diff.

**Comment by elezar** [comment]
I don't know what closed this ... reopening ...

**Pull request closed without merging**